### ISSUE TITLE
Voice Volume too low

### ISSUE THREAD

**Issue by Warri2k** [michail-nikolaev/task-force-arma-3-radio#1010]
Voice too low, even in yelling mode. Everything above ~20 meter not hearable. Even when close together it's really quiet.

**Comment by kavinsky** [comment]
When you starts the TS3 and connects to the server without starting the ArmA3 it sounds low too?

**Comment by ACyprus** [comment]
Are you running any earplug mods, eg ACE?

**Comment by macchky** [comment]
Direct voice too low me too! 
Radio Volume is fine, but direct is lower for some reason since around this year (I don't remenber correctly)

**Comment by rogerxxxx** [comment]
Ditto here too.

**Comment by Raspu86** [comment]
Are you guys using ace_hearing? Ace has the very annoying feature to reduce TFAR direct speech drastically if earplugs are in or if you wear some helmets.

**Comment by rogerxxxx** [comment]
Nope.  Just one of those common things with direct chat within the game overall.

I'm piping the games six channel sound through Toslink (ie. S/PDIF), into my Yamaha receiver having a 5 point surround sound system.  But users really do not need such a sound system to notice the overall low levels of direct chat.

I can't remember exactly, but I also think direct chat is monitored by the Effects volume control instead of the Radio volume control.

**Comment by dedmen** [comment]
Can i get some more info on that? Are people still having issues? We could just add a client setting which tells the Teamspeak plugin to increase the volume of direct speech.

**Comment by RaynorD** [comment]
Yes, this is still an issue. Ingame effects, namely aircraft, are extremely loud compared to TFR unless effects level is adjusted to 3-5%. Secondly if there's anyone in channel not in TFR, they are extremely loud compared to those who are.
Combined or preferably separate settings in TS for overall direct volume gain and overall radio volume gain would be extremely helpful.

**Comment by dedmen** [comment]
I can't really reproduce that.. In fact I'm in a mission right now.. and it seems realistic to me. Best would be if you could make a video as example

**Comment by Slugg087** [comment]
I have recorded a video where I have this problem: https://youtu.be/oKEZWMgfFDM?t=25s We use ACE among others, if you want a full list of the mods i can show you.

**Comment by dedmen** [comment]
@Slugg087 Video not available. If using ACE make sure its not because of earplugs

**Comment by Slugg087** [comment]
@dedmen Changed it to unlisted should work now.

**Comment by dedmen** [comment]
Those guys in the distance at 0:45 sound a bit too loud
at 2:10 yeah.. sounds a bit too silent but they are calmly talking in normal volume.. so its about right
The constant white noise in the background is killing me ^^
I'd really like to make the volume configurable.. But ace is using the globalVolume variable for deafness, unconciousness and earplugs.. And will constantly change it and reset it to maximum of 1 when no effects apply.. So we would probably need another global volume setting.

**Comment by Slugg087** [comment]
Yeah I think it shows better at 08:55 where we talk in game first then the server restarts and we talk in TS nomally. Then you can hear a big difference in clarity and volume. Ok should i talk to the developers of ACE and see if they can fix it?

**Comment by dedmen** [comment]
No.. I could fix ACE myself but.. There is more behind that value than just TFAR's volume.
Well you have to keep in mind that you are standing a few meters apart. Where in normal Teamspeak the other guys are directly inside your ear.

**Comment by RaynorD** [comment]
Raising the curve will help, but in my opinion an overall gain adjustment, configurable or permanent, is badly needed to equalize tfr vs non-tfr clients' levels. In real life there's no one directly inside your ears that you don't intend to have there, usually with a volume knob. ;)
